src/OpenSage.Tools.BigEditor/UI/MainForm.cs

```diff
@@ -15,15 +16,21 @@ internal sealed class MainForm : DisposableBase
         private BigArchive _bigArchive;
         private readonly List<BigArchiveEntry> _files;
         private int _currentFile;
+        private string _currentFileName;
         private string _currentFileText;
 
         private byte[] _searchTextBuffer;
         private byte[] _filePathBuffer;
         private string _searchText;
+        static float _scrollY;
```

Comment:
I don't think this needs to be `static`.

Reply:
Fixed